submarine-server/server-core/src/main/java/org/apache/submarine/server/rest/JobManagerRestApi.java

```diff
@@ -19,32 +19,32 @@
 
 package org.apache.submarine.server.rest;
 
-import org.apache.submarine.server.JobManager;
-import org.apache.submarine.server.api.exception.UnsupportedJobTypeException;
-import org.apache.submarine.server.api.job.Job;
-import org.apache.submarine.server.api.job.JobId;
-import org.apache.submarine.server.api.spec.JobSpec;
-import org.apache.submarine.server.response.JsonResponse;
-
 import javax.ws.rs.Consumes;
 import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
+import javax.ws.rs.PATCH;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.submarine.server.job.JobManager;
+import org.apache.submarine.server.api.job.Job;
+import org.apache.submarine.server.job.JobServiceException;
+import org.apache.submarine.server.api.spec.JobSpec;
+import org.apache.submarine.server.response.JsonResponse;
+
 /**
- * {@link JobManager}'s REST API v1. It can accept {@link JobSpec} to create a job.
+ * Job Service REST API v1. It can accept {@link JobSpec} to create a job.
  */
 @Path(RestConstants.V1 + "/" + RestConstants.JOBS)
 @Produces({MediaType.APPLICATION_JSON + "; " + RestConstants.CHARSET_UTF8})
 public class JobManagerRestApi {
-
+  private final JobManager service = JobManager.getInstance();
```

Comment:
Please use a more meaningful name like "jobManager" instead of "service", the name "service" is unclear.